packages/contracts/contracts/L1/AssetManager.sol

```diff
@@ -396,16 +417,21 @@ contract AssetManager is ISemver, IERC721Receiver, IAssetManager {
     /**
      * @inheritdoc IAssetManager
      */
-    function initUndelegate(address validator, uint128 shares) external {
-        if (shares == 0) revert NotAllowedZeroInput();
+    function undelegate(address validator, uint128 assets) external {
```

Comment:
Please modify the interface of this function!

Reply:
Oops, thanks